src/Nemo.jl

```diff
@@ -251,7 +251,16 @@ function __init__()
    ccall((:flint_set_abort, libflint), Nothing,
          (Ptr{Nothing},), @cfunction(flint_abort, Nothing, ()))
 
-   if isinteractive() &&
+   # Check if were are non-interactive
+   io = IOBuffer()
+   Base.show_backtrace(io, Base.backtrace())
+   bt = String(take!(io))
+   isinteractive_manual = !occursin("_tryrequire_from_serialized", bt)
```

Comment:
Not really faster, but a tad shorter and less allocations:
```
   bt = Base.process_backtrace(Base.backtrace())
   isinteractive_manual = all(sf -> sf[1].func != :_tryrequire_from_serialized, bt)
```

Reply:
Did you try this on all julia versions? Just to make sure that the `.func` field was not renamed.

Reply:
All except version 1.3 which I don't have anymore locally.